### ISSUE TITLE
[MacOS] pod install can't find file on githubusercontent.com

### ISSUE THREAD

**Issue by RomainQuidet** [CocoaPods/CocoaPods#10493]
### Command

```
/Users/romainquidet/.gem/bin/pod install
```

### Report

* What did you do? Run pod install

* What did you expect to happen? pods to be installed

* What happened instead? ruby crash

### Stack

```
   CocoaPods : 1.10.1
        Ruby : ruby 3.0.0p0 (2020-12-25 revision 95aff21468) [x86_64-darwin19]
    RubyGems : 3.2.3
        Host : Mac OS X 10.15.7 (19H524)
       Xcode : 12.4 (12D4e)
         Git : git version 2.30.1
Ruby lib dir : /usr/local/Cellar/ruby/3.0.0_1/lib
Repositories : accengage - git - https://github.com/Accengage/ios-pod-specs.git @ 4812775500561c12b0432c7bb0f1088415ea7128

               cocoapods - git - https://github.com/CocoaPods/Specs.git @ 8487c11833b003f182bfc9332914205e8f963468

```

### Plugins

```
cocoapods-clean       : 0.0.1
cocoapods-deintegrate : 1.0.4
cocoapods-plugins     : 1.0.0
cocoapods-search      : 1.0.0
cocoapods-trunk       : 1.5.0
cocoapods-try         : 1.2.0
```

### Podfile

```ruby
source 'https://github.com/CocoaPods/Specs.git'
source 'https://github.com/Accengage/ios-pod-specs.git'

platform :ios, '11.0'

use_frameworks!

def shared_pods
  pod 'Firebase/Core'
  pod 'Firebase/RemoteConfig'
  pod 'Firebase/Performance'
  pod 'FirebaseInstallations'
  pod 'Firebase/Crashlytics'
  pod 'Firebase/Analytics'
  pod 'Alamofire'
  pod 'RxSwift'
  pod 'RxCocoa'
  pod 'RxAlamofire'
  pod 'ObjectMapper'
  pod 'RxDataSources'
  pod 'SwiftKeychainWrapper'
  pod 'CryptoSwift', '1.3.0'
  pod 'Accengage-iOS-SDK', '6.3.1'
  #TC libs for debug
  pod 'latest-TCCore-Debug-universal', :podspec => 'https://raw.githubusercontent.com/TagCommander/pods/master/TCCore/latest-TCCore-Debug-universal.podspec'
  pod 'latest-TCSDK-Debug-universal', :podspec => 'https://raw.githubusercontent.com/TagCommander/pods/master/TCSDK/latest-TCSDK-Debug-universal.podspec'
  pod 'IQKeyboardManagerSwift', '6.1.1'
  pod 'lottie-ios', '3.0.4'
  pod 'pop'
  pod 'IdentitySdkCore', '5.2.2'
  pod 'IdentitySdkWebView', '5.2.2'
  pod 'IdentitySdkGoogle', '5.2.2'
  pod 'IdentitySdkFacebook', '5.2.2'
  pod 'FBSDKShareKit'
  pod 'IKEventSource'
  pod 'iCarousel', '~> 1.8'
  pod 'PayCardsRecognizer'
  pod 'Usabilla'
  pod 'PromisesSwift'
  pod 'Dimelo-iOS', :git => 'https://github.com/ringcentral/engage-digital-messaging-ios.git'
  pod 'Airship', '14.1'
  pod 'TTTAttributedLabel'
  pod 'SwiftyGif'
end

target 'Particuliers-Stubs' do
    shared_pods
end
```

### Error

```
Errno::ENOENT - No such file or directory @ rb_sysopen - https://raw.githubusercontent.com/TagCommander/pods/master/TCCore/latest-TCCore-Debug-universal.podspec
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/external_sources/podspec_source.rb:19:in `initialize'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/external_sources/podspec_source.rb:19:in `open'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/external_sources/podspec_source.rb:19:in `block in fetch'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:86:in `titled_section'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/external_sources/podspec_source.rb:11:in `fetch'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:989:in `fetch_external_source'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:968:in `block (2 levels) in fetch_external_sources'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:967:in `each'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:967:in `block in fetch_external_sources'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:966:in `fetch_external_sources'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer/analyzer.rb:117:in `analyze'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:414:in `analyze'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:239:in `block in resolve_dependencies'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/user_interface.rb:64:in `section'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:238:in `resolve_dependencies'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/installer.rb:160:in `install!'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/command/install.rb:52:in `run'
/Users/romainquidet/.gem/gems/claide-1.0.3/lib/claide/command.rb:334:in `run'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/lib/cocoapods/command.rb:52:in `run'
/Users/romainquidet/.gem/gems/cocoapods-1.10.1/bin/pod:55:in `<top (required)>'
/Users/romainquidet/.gem/bin/pod:23:in `load'
/Users/romainquidet/.gem/bin/pod:23:in `<main>'
```

**Comment by soober** [comment]
Is there any progress on the question?

**Comment by dnkoutso** [comment]
Its possible it does not have access to the file...

**Comment by guillaumealgis** [comment]
Looks like the same issue as https://github.com/CocoaPods/CocoaPods/issues/10597

**Comment by dnkoutso** [comment]
this was fixed for 1.11.0.

**Comment by dnkoutso** [comment]
going to close for now.
